device/bgp/bgp.yaml

```diff
@@ -25,8 +45,11 @@ components:
             Internal BGP is used within a single autonomous system (ibgp).
             BGP property defaults are aligned with this object defined as an
             internal BGP peer.
-            If the as_type is specified as ebgp then other properties will
-            need to be specified according an external BGP peer.
+            If the as_type is specified as 'ebgp' then other properties will
```

Comment:
Missing full stop in the description.

`External BGP is used for BGP links between two or more autonomous systems (ebgp).`